### PR TITLE
Move Google.NearbyShare 1.0.993.0 to Google.QuickShare 1.0.993.0

### PR DESCRIPTION
**Pull request by SpecterShell**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/146157)

### CODE REVIEW

**Review by mdanish-kh — APPROVED** [review]

**Review by mdanish-kh — APPROVED** [review]

**Comment by mdanish-kh** [comment]
https://github.com/microsoft/winget-pkgs/issues/142854

File: manifests/g/Google/QuickShare/1.0.993.0/Google.QuickShare.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+AppsAndFeaturesEntries:
+- ProductCode: '{DA6F3C20-298A-43EE-AA8D-6BABFC0458B0}'
+  UpgradeCode: '{D23F687E-D500-4BFF-8C2E-D3417ECE5F27}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://dl.google.com/release2/Nearby/acyrqme3i6smszcjbr5fyot54xqa_1.0.993.0/better_together.msi
+  InstallerSha256: EFBD1E6064FC7544C37033F9F382F9A53480A7C9E1BFB2F3BCB9C653F833F22A
+  ProductCode: '{DA6F3C20-298A-43EE-AA8D-6BABFC0458B0}'
+ManifestType: installer
+ManifestVersion: 1.4.0
```
Comment (stephengillie) [inline manifests/g/Google/QuickShare/1.0.993.0/Google.QuickShare.installer.yaml:20]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/g/Google/QuickShare/1.0.993.0/Google.QuickShare.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+- send
+- share
+- transfer
+# ReleaseNotes:
+# ReleaseNotesUrl:
+# PurchaseUrl:
+# InstallationNotes:
+# Documentations:
+ManifestType: defaultLocale
+ManifestVersion: 1.4.0
```
Comment (stephengillie) [inline manifests/g/Google/QuickShare/1.0.993.0/Google.QuickShare.locale.en-US.yaml:34]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/g/Google/QuickShare/1.0.993.0/Google.QuickShare.locale.zh-CN.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+- 安卓
+- 文件
+- 附近分享
+# ReleaseNotes:
+# ReleaseNotesUrl:
+# PurchaseUrl:
+# InstallationNotes:
+# Documentations:
+ManifestType: locale
+ManifestVersion: 1.4.0
```
Comment (stephengillie) [inline manifests/g/Google/QuickShare/1.0.993.0/Google.QuickShare.locale.zh-CN.yaml:35]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/g/Google/QuickShare/1.0.993.0/Google.QuickShare.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.2.8 $debug=QUSU.CRLF.7-3-6.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Google.QuickShare
+PackageVersion: 1.0.993.0
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.4.0
```
Comment (stephengillie) [inline manifests/g/Google/QuickShare/1.0.993.0/Google.QuickShare.yaml:8]:
```suggestion
ManifestVersion: 1.6.0
```

**Review by stephengillie — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**